content/blog/scaling-nats-for-education.md

```diff
@@ -0,0 +1,97 @@
+
+So where does the technology come in? Well, first of all we're not a product house. We don't create things that you can buy, or provide services that you can run.
+
+Where we intersect in the technology space is to identify those structural gaps that need tools to fill them, to create those tools as open source components, and then make them available to all our stakeholders to use in solving problems; especially problems in the integration space. Integration problems are some of the hardest to address structurally since it can often be the case that it's not clear in a classic market relationship quite who has responsibility for solving them - vendors rightly will concetrate on putting new features into their product sets, but who creates the links to other packages or data sources, and what's the incentive to do so.
+
+We can't provide any direct incentives, but what we can do is create open source tools to solve those problems and then make them available to everyone to embed or use as easily as possible.
+
+Our latest toolset is a suite of services that provide a whole bunch of useful integration tools. In the popular language of today you could call it a micro-services architecture. My only hesitation in actually calling it that is that I feel increasingly like the Will Munny character in 'Unforgiven' - "I've sent messages across every protocol that walks or crawled at one time or another" - I fell in love with message-based architectures a long time ago, have always felt them to be inherently helpful in dealing with complex problems, and here we all are in the wondeful world of reactive systems. 
+
+I'm truly glad that the rest of the world has caught up with us queue-junkies, but sometimes with a little bit of that feeling when the obscure band you loved suddenly gets famous and everybody else loves them too. Okay, enough with grizzled neck-beard nostalgia - if everyone loves messaging that's all to the good with me.
+
+Our toolset is known as NIAS (NSIP Integration as A Service), and I want to talk about one particular subset of those services and how NATS has helped us.
+
+Each component of NIAS is a standalone utility that does something useful, and all of the services communicate over a messaging backbone; NATS and soon STAN as well.
+
+There's a whole bunch of these services; Privacy Filtering, a small 'graph' datastore that links entities from different inbound data models, lightweight analytics, and the set I want to focus on here that specialise in data validation.
+
+So here's our validation scenario. Schools and school authorities need to register students for online national assessment. For that to work well, there's a whole set of business conditions that need to be tested. Are students in the right year groups for the  tests based on their birthdates, are there duplicate records for students, are they enrolled in more than one school, does the supllied data match the required schema for structure and content etc. etc.
```

Comment:
These may be good broken out as bulleted list? Just a thought - but the examples make total sense.